packages/replay/src/client.ts

```diff
@@ -96,6 +121,8 @@ class ProtocolClient {
       this.pendingMessages.delete(msg.id);
       if (msg.result) {
         resolve(msg.result);
+      } else if (msg.error) {
+        reject(new ProtocolError(msg.error));
```

Comment:
After a second though, it might be best to **temporarily** change this to:
```suggestion
        reject(msg.error);
```

Even though I feel like the current change is better, I can't quickly analyze how it might affect all the other callers. There is a really high chance that it wouldn't affect them negatively (or at least not in a disastrous way :P) but we never know.

WDYT?

Reply:
I don't think it would impact in any negative way.